CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# ksh x.y.z (version TBD, this is a work in progress)
+
+Starting in May 2017 maintenance of the Korn shell (`ksh`) resumed with
+the merging of some fixes from RedHat by Siteshwar Vashisht. In October
+2017 Kurtis Rader noticed that `ksh` had been open sourced and started
+contributing changes. This document was subsequently created to document
+the work being done. Whether the next stable version is considered a
+minor or major release is TBD.
+
+## Deprecations
+
+None at this time.
+
+## Notable non-backward compatible changes
+
+- Support for the UWIN environment has been removed (issue #284).
+- The `SHOPT_FIXEDARRAY` feature has been removed. This should not break any
```

Comment:
`SHOPT_FIXEDARRAY` was enabled in Fedora.

Reply:
How likely is it anyone is using that feature? See issue #234 for why I am skeptical that anyone is using it and why it shouldn't be used.